services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -1066,6 +1068,123 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         flow && !deleteResponse && flowHelperV2.deleteFlow(flow.flowId)
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/3049")
+    @Tidy
+    def "Able to update a flow endpoint"() {
+        given: "Three active switches"
+        def allSwitches = topology.activeSwitches
+        assumeTrue("Unable to find three active switches", allSwitches.size() >= 3)
+        def srcSwitch = allSwitches[0]
+        def dstSwitch = allSwitches[1]
+
+        and: "A vlan flow"
+        def flow = flowHelperV2.randomFlow(srcSwitch, dstSwitch, false)
+        flowHelperV2.addFlow(flow)
+
+        when: "Update the flow: port number and vlan id on the src endpoint"
+        def flowInfoFromDb1 = database.getFlow(flow.flowId)
+        def newPortNumber = 10 // this port is always available
+        def newVlanId = flow.destination.vlanId
```

Comment:
this is fine I guess, but I don't really understand what is behind this. src/dst vlans may be equal

Reply:
agree, changed